src/components/narrative/MobileNarrativeDisplay.js

```diff
@@ -127,6 +127,19 @@ class MobileNarrativeDisplay extends React.Component {
           </MobileContentContainer>
         </>
       );
+    } else if (this.props.currentInFocusBlockIdx === 0) {
+      return (
+        <>
+          <MobileContentContainer height={contentHeight}>
+            {this.pageNarrativeContent()}
```

Comment:
block 0 is the title page & shouldn't display viz / main-markdown (see sketch file for more notes if needed). It'll probably need special styling too to correctly display the header / authors etc.

Reply:
Is this still relevant? I don't know that I fully understand what you're saying here. Sorry I missed it earlier.